### PR TITLE
WIP README.md

### PR DESCRIPTION
**Pull request by zamiraskaff**
WIP readme for BE

Let me know if the GraphQL endpoints are correctly represented, as well as any other feedback or additions

### CODE REVIEW

**Review by amikaross — APPROVED** [review]
This looks great! I'll let others review before merging, but i don't have any comments atm.  Edit: Actually just thought of something - I know you post the link to the heroku url at the top of the readme but we should probably add the path for the graphql types/mutations down in the graphql section as well.

**Review by naomiyocum — APPROVED** [review]
This looks awesome! Thank you for setting this up

**Pull request merged**
